sootup.analysis/src/main/java/sootup/analysis/interprocedural/icfg/CGEdgeUtil.java

```diff
@@ -59,12 +60,11 @@ public static Set<Pair<MethodSignature, CalleeMethodSignature>> getCallEdges(
       SootMethod method = view.getMethod(caller).orElse(null);
       if (method != null && method.hasBody()) {
         for (Stmt s : method.getBody().getStmtGraph().getNodes()) {
-          if (s.containsInvokeExpr()) {
+          if (s instanceof InvokableStmt && ((InvokableStmt) s).containsInvokeExpr()) {
```

Comment:
Not all Invokable stmt contain a Invoke expr. These are not the same. Even doesInvoke is not the same since "int a = 5;" would be also an InvokableStmt which does not invoke anything.
And I thought we wanted to go away from the stmt thingy that throws exceptions for not fitting stmts. On the other side you could say it could be weird for the developer that all other stmts throw an exception?